### ISSUE TITLE
Crash issue with OCExpandableButtonAlignmentRight

### ISSUE THREAD

**Issue by fabiomassimo** [ocrickard/OCExpandableButton#2]
When I try to use OCExpandableButtonAlignmentRight blocks simulator with strange animation behaviour .

**Comment by ocrickard** [comment]
Which iOS simulator version?  Is this the demon project?  Sample code?

**Comment by fabiomassimo** [comment]
On latest iOS simulator version. On my project and on example project.
On example project, I've changed:

```
//You can change the alignment with:
button.alignment = OCExpandableButtonAlignmentRight;
```

When I build and run something goes nuts.

**Comment by ocrickard** [comment]
Thanks. I'll take a look when I get to work.

**Comment by ocrickard** [comment]
OK, so it turns out that the directionModifier was an integer, forcing type casts to integer for certain angles and alignments during animation.  Fixed in dcb96af3032686fce0017130f3dadb44e2e324c0.
